x/marker/keeper/keeper_test.go

```diff
@@ -943,3 +946,40 @@ func TestGetAuthority(t *testing.T) {
 	app := simapp.Setup(t)
 	require.Equal(t, "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn", app.MarkerKeeper.GetAuthority())
 }
+
+func TestRemoveIsSendEnabledEntries(t *testing.T) {
+	app := simapp.Setup(t)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+	markerAddr := markertypes.MustGetMarkerAddress("testcoin").String()
+
+	err := app.MarkerKeeper.AddMarkerAccount(ctx, &markertypes.MarkerAccount{
+		BaseAccount: &authtypes.BaseAccount{
+			Address:       markerAddr,
+			AccountNumber: 23,
+		},
+		AccessControl: []markertypes.AccessGrant{
+			{
+				Address:     "cosmos1w6t0l7z0yerj49ehnqwqaayxqpe3u7e23edgma",
+				Permissions: markertypes.AccessListByNames("deposit,withdraw"),
+			},
+		},
+		Denom:      "testcoin",
+		Supply:     sdk.NewInt(1000),
+		MarkerType: markertypes.MarkerType_Coin,
+		Status:     markertypes.StatusActive,
+	})
+	require.NoError(t, err, "should have added marker")
+	app.BankKeeper.SetSendEnabled(ctx, "testcoin", false)
+	app.BankKeeper.SetSendEnabled(ctx, "nonmarkercoin", false)
+
+	sendEnabledItems := app.BankKeeper.GetAllSendEnabledEntries(ctx)
+	assert.Equal(t, len(sendEnabledItems), 2, "should have 2 items before removal")
```

Comment:
When doing slice length assertions, the `.Len(...)` assertion is the way to go.
```suggestion
	assert.Len(t, sendEnabledItems, 2, "before removal")
```

The nice thing there is that when the assertion fails, the contents of `sendEnabledItems` will be in the failure message.

Even better would be asserting things exactly using `.Equals`. If you don't want to worry about ordering, you can use `ElementsMatch`. If the `actual` might have extra things, and you just want to make sure your things exist, you can use `.Contains` once for each expected entry.

But at the very least, please use `.Len` instead of `.Equals(t, len(...), ...)`.

Reply:
Thanks, didn't know this existed or think to look.